app/src/main/java/org/eyeseetea/malariacare/layout/adapters/survey/DynamicTabAdapter.java

```diff
@@ -111,7 +109,7 @@
     /**
      * Listener that detects taps on buttons & swipe
      */
-    private OnSwipeTouchListener swipeTouchListener;
+    private SwipeTouchListener swipeTouchListener;
```

Comment:
@idelcano why do you change this? You kept OnSwipeTouchListener class but it looks to me that it is never used, right?